GoMeet/src/main/java/com/google/sps/dao/LocationDao.java

```diff
@@ -59,11 +62,15 @@ public LocationDao() {
    * Returns the key string of the new entity.
    */
   @Override
-  public String save(Location location) throws SimilarEntityExistsException {
+  public String save(Location location) throws MaxEntitiesReachedException, SimilarEntityExistsException {
```

Comment:
Optional: It might be good to explain why the case where the maximum number of locations are reached is handled exclusively in the back-end? 
Because there could potentially also be counters in the front-end to see how many locations a user proposes.

Reply:
Yup, the explanation is that:

One meeting can have a maximum of 5 locations. If two people are trying to add a location at the same time, there could be the case that:
- Person A and Person B open the event page. If they had front-end counters, it would be at 4. 
- Person A sends in a new location and it gets added to the meeting. Their counter is now 5 but person B's counter is still 4. 
- Person B sends in a new location since their counter still says 4. If there was no server side check, then the meeting would have 6 locations in total. 

I could not implement real-time updating as that was a stretch goal.